stage1/init/kvm/network.go

```diff
@@ -158,7 +158,9 @@ func GenerateNetworkInterfaceUnits(unitsPath string, netDescriptions []netDescri
 			return errwrap.Wrap(fmt.Errorf("failed to create network unit file %q", unitName), err)
 		}
 
-		rlog.Printf("network unit created: %q in %q (iface=%q, addr=%q)", unitName, unitsPath, ifName, address)
+		if debug {
```

Comment:
What do you mean by "not get passing through"? In SetDebug?
With commented `if` everything seems working fine, so probably you have in mind a problem on logger side (and this is probably a reason for rest of messages described in #2622), am I correct?